src/scss/tools/_typography.scss

```diff
@@ -89,11 +89,18 @@
 
 @mixin input-label {
   font-size: 14px;
-  line-height: 20px;
```

Comment:
I think remove this line is going to change the line-height of all the current input's labels from 20px to 24px. Is that what we really want?